Refactor unit system, implement reciprocal unit conversion (#1025)
dftbplus/dftbplus: doc/dftb+/manual/unit_mod.tex

below.\footnote{The conversion factors listed here were calculated with double
  precision on i686-linux architecture.  Depending on your architecture, the
  values used there may deviate slightly.}  (The modifiers are case
insensitive). The symbol $\left[x^{-1}\right]$ indicates conversions, where

> not the value, but the reciprocal value is multiplied by the conversion

Reviewer: ```suggestion
not the value, but the reciprocal of the quantity is used in the conversion
```
